src/python/WMCore/Storage/Backends/GFAL2Impl.py

```diff
@@ -24,9 +24,9 @@ def __init__(self, stagein=False):
         # Next commands after separation are executed without env -i and this leads us with 
         # mixed environment with COMP and system python.
         # GFAL2 is not build under COMP environment and it had failures with mixed environment.
-        self.setups = "env -i X509_USER_PROXY=$X509_USER_PROXY sh -c '%s'"
-        self.removeCommand = self.setups % 'printenv; date; gfal-rm -vvv -t 600 %s '
-        self.copyCommand = self.setups % 'printenv; date; gfal-copy -vvv -t 2400 -T 2400 -p %(checksum)s %(options)s %(source)s %(destination)s'
+        self.setups = "env -i X509_USER_PROXY=$X509_USER_PROXY bash -c '%s'"
+        self.removeCommand = self.setups % '. $PWD/startup_environment.sh; printenv; date; gfal-rm -vvv -t 600 %s '
+        self.copyCommand = self.setups % '. $PWD/startup_environment.sh; printenv; date; gfal-copy -vvv -t 2400 -T 2400 -p %(checksum)s %(options)s %(source)s %(destination)s'
```

Comment:
In production test, $PWD is not the same location which startup_environment.sh file is created.
Is CRAB put the file in different location than wmagent?

```
/var/lib/condor/execute/dir_3127117/glide_COk0nT/execute/dir_3145575/job/WMTaskSpace/stageOut1/startup_environment.sh: No such file or directory

actual file is located at 
/var/lib/condor/execute/dir_3127117/glide_COk0nT/execute/dir_3145575/
```